main.py

```diff
@@ -2,6 +2,47 @@
 
 from django.db.models import QuerySet
 
+from db.models import Actor, Genre
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
```

Comment:
assigning variable names should be at the start of function
by that you'll improve code readability and ensure that variables are defined and initialized in a logical order, making it easier to understand and maintain the code.